### ISSUE TITLE
React complains about string values of `required` prop when decorator is used

### ISSUE THREAD

**Issue by aquilae** [formsy/formsy-react#76]
```
const MyComponent = withFormsy(function() {
  return <div>MyComponent</div>;
});

render(<MyComponent required="myCustomValidation" />); // React expects only bool value here
```
Pull Request #75

**Comment by MilosRasic** [comment]
Good catch. Merged.

**Comment by rkuykendall** [comment]
@aquilae This change has been released in 1.1.3.
